src/lib/aws/secret_manager.py

```diff
@@ -0,0 +1,45 @@
+import json
+import boto3
+from botocore.exceptions import ClientError
+from typing import Dict
+
+
+class SecretManagerClientException(Exception):
+    """Error while retrieving a secret from AWS Secrets Manager."""
+
+    pass
+
+
+class SecretManager:
+    """Manages interactions with AWS Secrets Manager."""
+
+    def __init__(self, region_name: str, secret_client=None) -> None:
+        """Initiate class SecretManager.
+
+        Args:
+            secret_client: Boto3 SES client for AWS interactions.
+        """
+        self._secret_client = (
+            boto3.client(service_name="secretsmanager", region_name=region_name)
```

Comment:
Here, also, I'm not sure we need to use region_name param here

Reply:
Done.